packages/core-contracts/contracts/utils/ERC20.sol

```diff
@@ -0,0 +1,101 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../interfaces/IERC20.sol";
+
+abstract contract ERC20 is IERC20 {
+    string public override name;
```

Comment:
I don't looooooove to have to override a variable that was never _directly_ declared in the interface... but well, fair I guess.